basefunctions.hxx

```diff
@@ -14,6 +14,29 @@ enum Channel { MT = 0, ET = 1, TT = 2, EM = 3 };
 
 namespace basefunctions {
 
+/// This function filters events, where neither of the input flags is true.
+/// This is used to filter events which do not pass an underlying requirement in
+/// any systematic variation.
+///
+/// \param df The input dataframe
+/// \param filtername The name of the filter, used in the Dataframe report
+/// \param flags Parameter pack of column names that contain the considered
+/// flags
+///
+/// \returns a filtered dataframe
+template <class... Flags>
+auto FilterPassAny(auto df, const std::string filtername,
```

Comment:
Maybe change name to `FilterFlags` - in conjunction with `flagGoodPairs`

Reply:
maybe FilterFlagsAny or FilterFlagsOR?

Reply:
i like `FilterFlagsAny` :+1: